psi4/src/export_fock.cc

```diff
@@ -127,7 +127,10 @@ void export_fock(py::module &m) {
         .def("current_IFock", &SOMCSCF::current_IFock)
         .def("zero_redundant", &SOMCSCF::zero_redundant)
         .def("gradient", &SOMCSCF::gradient)
-        .def("gradient_rms", &SOMCSCF::gradient_rms);
+        .def("gradient_rms", &SOMCSCF::gradient_rms)
+        .def("noccpi", &SOMCSCF::noccpi)
+        .def("nactpi", &SOMCSCF::nactpi)
+        .def("nvirpi", &SOMCSCF::nvirpi);
```

Comment:
These can be acquired from the `CIWavefunction::get_dimension` function as well.